www/js/controllers.js

```diff
@@ -47,23 +47,47 @@ module.controller('AppCtrl', function(pushService, $rootScope, $scope, $ionicMod
     // Triggered in the login modal to close it
     $scope.closeLogin = function() {
         $scope.modal.hide();
-        $scope.modal.remove();
+        //$scope.modal.remove();
     };
 
     // Open the login modal
     $scope.login = function() {
       $scope.modal.show();
     };
-
+	
+	$scope.logout = function() {
+		$localStorage.set("leaderAPIKey", "");
+	};
+
+	loginSuc = function(response) {
+		var data = response.data;
+		console.log(response);
+		console.log(data);
+		if(data.success) {
```

Comment:
I have been adding alerts like this here and there - if someone accepts this PR as is, I will add such an alert.